.circleci/images/build/Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM node:10.15.3-jessie
+FROM resinci/npm-x86_64-ubuntu-node10
 
-# install solc binary
+# add ethereum ppa
 RUN add-apt-repository ppa:ethereum/ethereum
+
 RUN apt-get update
+
 RUN apt-get install -y solc
-RUN node -v & solc --version
+
+# install node 10.x
```

Comment:
You can run all 3 commands in the same `RUN` step to cache the result in one layer instead of 3 if the execution of these 3 is tightly coupled. Obviously not a blocker, just mentioning it.